### ISSUE TITLE
Schedule meetings with co-working spaces

### ISSUE THREAD

**Issue by dlindahl** [techramento/techramento.github.io#7]
Reach-out to the various co-working spaces in Sacramento to setup up an in-person meeting to discuss if they'd be interested in partnering with us. The idea is that we'd promote the space and bring in potential clients through meetups in exchange for having a reserved "Techramento" night. This reserved night would be available for any of our meetups to book. This would make it easier for meetups to find a reliable meetup space that is conducive, and agreeable to, hosting a recurring meetup.

Ideally we'd like to have at least two board members attend the meeting, but preferably four. This allows all of us to meet the co-working space leaders so we can build a relationship with them.

First Round:
- [ ] The Urban Hive
- [ ] Capsity
- [ ] The Trade

Second Round:
- [ ] Hacker Lab
- [ ] Outlet

**Comment by dlindahl** [comment]
Blocked on #8

**Comment by dlindahl** [comment]
Tabled to next meeting (in order to be tabled to the next meeting :smile:)

**Comment by dlindahl** [comment]
On hold until #33 is done.

**Comment by dlindahl** [comment]
@efletch will reach out to The Outlet with an informal email to lead into our Open Letter.

**Comment by dlindahl** [comment]
@roughani will share the Open Letter to Shaun and Ryan on July 5th, ask for feedback, and setup a meeting.

**Comment by dlindahl** [comment]
Closed in lieu of the decision made at #8
